packages/neuron-ui/src/types/Controller/index.d.ts

```diff
@@ -56,7 +56,7 @@ declare namespace Controller {
 
   interface SendTransactionParams {
     walletID: string
-    tx: string
+    tx: any
```

Comment:
Why set it `any` here?

Reply:
because if it is string type, i give errors in  `OfflineSignDialog` when yarn start:ui:

```
ERROR in src/components/OfflineSignDialog/index.tsx:99:47
TS2322: Type '{ [key: string]: any; signatures?: Signatures | undefined; }' is not assignable to type 'string'.
     97 |               break
     98 |             }
  >  99 |             await sendTransaction({ walletID, tx: transaction, description, password })(dispatch).then(({ status }) => {
        |                                               ^^
    100 |               if (isSuccessResponse({ status })) {
    101 |                 history.push(RoutePath.History)
    102 |               } else if (status === ErrorCode.PasswordIncorrect) {
```

There are also other components call `sendTransaction`, I thought the type of `tx` could also be string, so changed it to any. Not sure whether we can use more detailed type here..

Reply:
This type error exists in develop branch too, and is also declared as `any` in https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/src/types/App/index.d.ts#L69

We could fix it later.